src/workflows-app/SubmissionConfig.js

```diff
@@ -120,6 +125,32 @@ export const BaseSubmissionConfig = (
     [methodId, signal]
   );
 
+  const loadCbasSubmissionLimits = useCallback(
```

Comment:
This method and the places it is called below are the main changes in this PR. Most of the refactoring changes are to make SonarCloud happy.